docs/getting-started.md

```diff
@@ -54,73 +108,101 @@ Have a nice day! 👋
 
-### 4 - Start the GitOps Dashboard web UI
+To learn more about these files, see our [GitOps Automation](gitops-automation.md).
+
+
+## Configure Weave GitOps to deploy the podinfo application
+
+### 5 - Start the GitOps Dashboard web UI
+
+Weave GitOps provides a web UI to help manage the lifecycle management of your applications.
 
 ```
 gitops ui run
 ```
 
-This will open the dashboard in your browser at [http://0.0.0.0:9001/](http://0.0.0.0:9001/).
+Running the above command will open the dashboard in your browser at [http://0.0.0.0:9001/](http://0.0.0.0:9001/).
 
 You will see an empty Applications view as shown in the image below.
 
 ![GitOps Dashboard web UI - empty applications view](/img/dashboard-applications-empty.png)
 
-### 5 - Add the podinfo application
+### 6 - Add the podinfo application
 
 Click **add application** in the top right of the screen to bring up the following form:
 
-![GitOps Dashboard web UI - add applications form](/img/dashboard-add-application.png)
+![GitOps Dashboard web UI - add applications form](/img/dashboard-add-form-complete.png)
 
 Next fill out the form with the required values:
 
 - Name: **podinfo-deploy**
 - Kubernetes Namespace: **wego-system**  
-  -  **Leave as default**, this is where the automation objects for the application will be created.
-- Source Repo URL: (**your-forked-podinfo-repo**)
-  - For example: `https://github.com/sympatheticmoose/podinfo-deploy` or `git@github.com:sympatheticmoose/podinfo-deploy`
-- Config Repo URL: (**leave blank**)  
-  - It is **recommended** to use a separate repository as your config repo when deploying multiple applications, potentially to multiple clusters, however, for simplicity - you can also use the same repo as the `Source Repo URL`.
+  -  **Leave as default**, this is where the GitOps Automation objects for the application will be deployed.
+- Source Repo URL: (**`git@github.com:<username>/podinfo-deploy`**)
+  - URL references can be in either the HTTPS `https://github.com/sympatheticmoose/podinfo-deploy` or SSH `git@github.com:sympatheticmoose/podinfo-deploy` format.
+- Config Repo URL: (**`git@github.com:<username>/gitops-config`**)  
+  - Specify the same configuration repository from steps 1 and 4 (install).
 - Path: **./**  
-  - This allows you to specify a particular folder with a repository, should the repo contain more than a single application's deployment manifests.
+  - **Leave as default**, this allows you to specify a particular folder with a repository, should the repo contain more than a single application's deployment manifests.
 - Branch: (**should not require changing**)
 
-Click **Submit** in the bottom right of the form. This will result in an error as we have not recently authenticated with GitHub, so we are not yet authorized to raise a pull request against your repository.
-
-Click **Authenticate with GitHub** within the error message as shown below:
+Click **AUTHENTICATE WITH GITHUB** next to the `Source Repo URL` field.
```

Comment:
Small suggestion would be to add some dashes(or something similar) to help differentiate between user steps and info,  maybe like this:
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/3892786/144475098-aac801fd-6909-47b2-9604-c31b75f98316.png">